README.md

```diff
@@ -34,3 +34,11 @@ make dev-server
   attributes, which can be fixed by find-replaces across files. For example:
   * `xmlns:xlink` should be `xmlnsXlink`
   * `stroke-width` should be `strokeWidth`
+
+### Adding new Icons
+We use SVGs as JSX components for `Icon`s, following this process:
+1. Optimize the svg at [svgomg](https://jakearchibald.github.io/svgomg/)
+2. Make it React compatible with double quotes option at [svg2jsx](http://svg2jsx.herokuapp.com/)
```

Comment:
can we add a step 3 to go through and make sure any necessary DOM IDs and/classnames are prefixed with the name of the component to avoid collisions?
planning on scripting all of this, but while we're still doing it manually, we'll need to scan through